scipy/stats/stats.py

```diff
@@ -2821,6 +2821,25 @@ def iqr(x, axis=None, rng=(25, 75), scale=1.0, nan_policy='propagate',
     return out
 
 
+def _mad_1d(x, center, nan_policy):
+    # Median absolute deviation for 1-d array x.  No warning is generated
+    # if x is empty or all nan.
+    # x must be a 1-d numpy array.
+    # If nan_policy is not 'propagate', it is assumed to be 'omit'.
```

Comment:
note for posterity that the `nan_policy=='raise'` is supported inside `_contains_nan` so all 3 are covered